### PR TITLE
[Drip] New Components

### LINKED ISSUE #4996: Additional Drip Triggers
### App
Drip
https://developer.drip.com/

### Triggers
- Add an optional prop on the New Subscriber Added trigger to let users specify specific campaigns to filter the events on.
- Tag applied https://developer.drip.com/#subscriber-applied_tag
- Custom Event Performed https://developer.drip.com/#subscriber-performed_custom_event

### Actions
- Start someone on a workflow https://developer.drip.com/#start-someone-on-a-workflow

### PR DESCRIPTION
**Pull request by andrewjschuang**
**Actions**:

- Start someone on a workflow

**Triggers**:

- Tag applied
- Custom Event Performed
- Added an optional prop on the New Subscriber Added trigger to let users specify specific campaigns to filter the events on - @dannyroosevelt is this functionality supposed to be the **New Campaign Subscription Added** trigger?

Resolves #4996.

### CODE REVIEW

File: components/drip/sources/tag-applied/tag-applied.mjs [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import base from "../common/base.mjs";
+
+export default {
+  ...base,
+  key: "drip-tag-applied",
```
Comment (lcaresia) [inline components/drip/sources/tag-applied/tag-applied.mjs:5]:
```suggestion
  key: "drip-new-tag-applied",
```

**Review by lcaresia — CHANGES_REQUESTED** [review]
Thanks for your contribution!

Change `sources/tag-applied/tag-applied.mjs` to `sources/new-tag-applied/new-tag-applied.mjs`

**Review by lcaresia — APPROVED** [review]
LGTM!

**Comment by DilanAthukorala** [comment]
Verification completed, ready for release.

**Pull request merged**